source/blueprints/user/services.py

```diff
@@ -103,3 +104,15 @@ def verify_user_credentials(email: str, password: str) -> Union[User, None]:
         raise ValueError("Invalid credentials")
 
     return user
+
+def verify_password(current_password: str, new_password: str, confirm_password: str) -> str:
```

Comment:
Missing docstrings!

Reply:
Sorry man, i'll do it right nowm boss.

Reply:
If you forget the docstrings again you can ask for a song in Fantastico tv show!